packages/components/src/HeaderBar/HeaderBar.test.js

```diff
@@ -138,10 +134,12 @@ describe('HeaderBar', () => {
 			.dive()
 			.find('withI18nextTranslation(Intercom)');
 		expect(intercomTrigger.length).toBe(1);
-		expect(intercomTrigger.props()).toEqual({
-			className: 'btn btn-link',
-			id: 'my-intercom',
-			config: { app_id: 'e19c98d', vertical_padding: 70 },
+		expect(intercomTrigger.prop('className')).toEqual('btn btn-link');
+		expect(intercomTrigger.prop('id')).toEqual('my-intercom');
+		expect(intercomTrigger.prop('config')).toEqual({
+			app_id: 'e19c98d',
+			email: 'lol@lol.com',
+			vertical_padding: 70,
```

Comment:
I changed this assertion to multiple because `props()` now returns the element default props too. So let's only check the info we wanted too.